### PR TITLE
Added a new parameter "downstreamHealthcheck" to fileconfig

### PR DESCRIPTION
**Pull request by JamesYing**
Added a new parameter "downstreamHealthcheck" to fileconfig for down host health check

@TomPallister the build.ps1 file is merge from your latest version, I don't kown how ignore this file in pull request :(

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -158,15 +158,14 @@ if(-Not $SkipToolPackageRestore.IsPresent) {
     if((!(Test-Path $PACKAGES_CONFIG_MD5)) -Or
```
Comment (TomPallister) [inline build.ps1]: Please remove this file from review or just overwrite changes

Comment (JamesYing) [reply build.ps1]: sorry, I commit 6 files, how many files do you look at?

Comment (JamesYing) [reply build.ps1]: commit id 1b582a5e4c3a01f54717702d9f4c6eb3e2013cf5

Comment (TomPallister) [reply build.ps1]: @JamesYing I see 6 files but the PR does not have any functionality just adds downstreamHealthcheck to configuration which does not make sense if the system doesnt use it for something!

**Pull request closed without merging**